packages/icons/metadata.yml

```diff
@@ -336,6 +336,15 @@ icons:
         categories:
           - name: Actions
             subcategory: Navigation
+  - name: character-patterns
+    friendly_name: Character patterns
+    usage: This is a description for usage
+    categories:
+      - name: Organization
+        subcategory: Data
+    aliases:
```

Comment:
You gotta either specify aliases or leave this property off all together.

Reply:
removed that 'aliases' prop. Thanks Vince.